runtime/autoload/tar.vim

```diff
@@ -432,11 +433,14 @@ if v:shell_error != 0
   set nomod
 
   let &report= repkeep
+  exe "cd ".pwdkeep
+  silent exe "file tarfile::".escape_file
 endfun
 
 " ---------------------------------------------------------------------
 " tar#Write: {{{2
 fun! tar#Write(fname)
+  let pwdkeep = getcwd()
   let repkeep= &report
   set report=10
   " temporary buffer variable workaround because too fucking tired. but it works now
```

Comment:
And while we are here, it probably would be good to remove this strong expression.